src/pages/tBTC/Bridge/Minting/KnowledgebaseLinks.tsx

```diff
@@ -0,0 +1,61 @@
+import { StackProps, VStack } from "@chakra-ui/react"
+import { FC } from "react"
+import { ExternalHref } from "../../../../enums"
+import { useFetchDepositDetails } from "../../../../hooks/tbtc"
+import {
+  BridgeProcessResources,
+  BridgeProcessResourcesItemProps,
+} from "../components/BridgeProcessResources"
+import {
+  TransactionHistory,
+  TransactionHistoryItemType,
+} from "../components/TransactionHistory"
+import bitcoinJuiceIllustration from "../../../../static/images/bitcoin-juice.png"
+import { useTbtcState } from "../../../../hooks/useTbtcState"
+
+export type KnowledgebaseLinksProps = { depositKey?: string } & StackProps
+
+const resources: BridgeProcessResourcesItemProps[] = [
+  {
+    title: "Token Contract",
+    link: ExternalHref.btcConfirmations,
+  },
+  {
+    title: "Bridge Contract",
+    link: ExternalHref.mintersAndGuardiansDocs,
+  },
+  {
+    title: "Read out documentation",
+    description: "Everything you need to know about our contracts.",
+    link: ExternalHref.mintersAndGuardiansDocs,
+    variant: "expanded",
+    imageSrc: bitcoinJuiceIllustration,
+  },
+]
+
+export const KnowledgebaseLinks: FC<KnowledgebaseLinksProps> = ({
+  depositKey = "",
+  ...restProps
+}) => {
+  const { data } = useFetchDepositDetails(depositKey)
```

Comment:
I've actually mentioned this issue in another PR too: https://github.com/threshold-network/bitcoin-on-base/pull/37#discussion_r1511040829. Please take a look at it as we can combine the fix for both.

Reply:
@kpyszkowski I've actually mentioned this issue in another PR too: https://github.com/threshold-network/bitcoin-on-base/pull/37#discussion_r1511040829. Please take a look at it as we can combine the fix for both.

Reply:
That's how layout is expected to work - to separate main content from aside areas for separation of concerns. To accomplish it was mandatory to expose required data to be accessible outside the main content. 
The data required to render `KnowledgeBaseLinks` are `mintingRequestedTxHash` and `mintingFinalizedTxHash`. As you pointed out, it can be gathered by calling the hook that was previously called in `DepositDetails`, That way we don't have to keep it in global store. 
It it okay as is currently. I wouldn't like to change anything in this area.